internal/worker/bootstrap/worker.go

```diff
@@ -311,9 +311,21 @@ func (w *bootstrapWorker) seedInitialUsers(ctx context.Context) error {
 		},
 	})
 	// User already exists, we don't need to do anything in this scenario.
-	if errors.Is(err, usererrors.UserAlreadyExists) {
+	if errors.Is(err, accesserrors.UserAlreadyExists) {
 		return nil
 	}
+
+	err = w.cfg.UserService.AddExternalUser(
+		ctx,
+		permission.EveryoneUserName,
+		permission.EveryoneUserName.Name(),
```

Comment:
We don't add a display name for other external users when we add them implicitly, so we should be consistent here.

Reply:
Removed it from here